saleor/graphql/schema.graphql

```diff
@@ -4083,6 +4083,18 @@ enum PostalCodeRuleInclusionTypeEnum {
   EXCLUDE
 }
 
+type PreorderData {
+  isPreorder: Boolean!
+  globalThreshold: Int
+  globalSoldUnits: Int
+  endDate: DateTime
```

Comment:
Are all of these fields nullable? Just making sure that we have correct types, it makes life of frontend devs easier ;)

Reply:
- `globalThreshold` should be nullable - user can just not set it, according to spec.
- `endDate` should be nullable - user can just not set it and end preorder manually.
- `globalSoldUnits` - I'm not sure now, it's probably my typo during editing the tech spec. Number of sold units sounds like something that should not be nullable. Moreover, similar field for channel listing is not nullable. I will change it to `Int!`. cc @jwm0